mitosheet/src/components/taskpanes/UpdateImports/UpdateImportsPostReplayTaskpane.tsx

```diff
@@ -117,8 +117,14 @@ const UpdateImportsPostReplayTaskpane = (props: UpdateImportPostReplayTaskpanePr
                             if (_invalidImportIndexes === undefined) {
                                 return;
                             }
+
                             props.setInvalidImportMessages(_invalidImportIndexes);
 
+                            void props.mitoAPI.log('get_test_import_results', {
```

Comment:
Do we want a `clicked_update_from_post_replay_update_imports` log here as well?

Reply:
Sure